exploit/index.html

```diff
@@ -63,6 +63,6 @@ <h1 class="intro__title">
 
     <button id="test" class='largeButton' onclick="window.minmain();">Test</button>
 
-    <script src="minmain.js"></script>
+    <script src="minmain_1.0.0-4.1.0.js"></script>
```

Comment:
We can probably have the server send a different minmain depending on what UA it gets.

Reply:
Yes, that's what I was thinking about, but wanted feedback before introducing those changes.